src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -18,14 +20,19 @@ public class Hardware {
 			DRIVEBASE_FRONT_RIGHT_ENCODER_PORT = 6,
 			DRIVEBASE_BACK_LEFT_ENCODER_PORT = 9,
 			DRIVEBASE_BACK_RIGHT_ENCODER_PORT = 12;
-	public static final Rotation2d DRIVEBASE_FRONT_LEFT_ENCODER_OFFSET =
-			Rotation2d.fromDegrees(337.236);
-	public static final Rotation2d DRIVEBASE_FRONT_RIGHT_ENCODER_OFFSET =
-			Rotation2d.fromDegrees(251.982);
-	public static final Rotation2d DRIVEBASE_BACK_LEFT_ENCODER_OFFSET =
-			Rotation2d.fromDegrees(205.839);
-	public static final Rotation2d DRIVEBASE_BACK_RIGHT_ENCODER_OFFSET =
-			Rotation2d.fromDegrees(311.396);
+	// Offsets, first value is BONK and second value is competition robot
```

Comment:
Minor but is there any reason we couldn't define a private `isComp` like we do in drivebase and use a ternary operator off of that? (`... ENCODER_OFFSET = isComp? Rotation2d.fromDegrees(...) : Rotation2d.fromDegrees(...);`)
(Even more minor but using a static initializer for `GEAR_RATIO` does nothing)